executor/inspection_result.go

```diff
@@ -148,6 +151,9 @@ func (e *inspectionResultRetriever) retrieve(ctx context.Context, sctx sessionct
 		}
 		// make result stable
 		sort.Slice(results, func(i, j int) bool {
+			if (results[i].degree > 0 || results[j].degree > 0) && results[i].degree != results[j].degree {
```

Comment:
If the `degree default to `0`, we can simplify the expression as `if results[i].degree != results[j].degree {`